bin/uninstall-cf.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Delete CF for Kubernetes
+kapp delete -a cf -y
```

Comment:
In fact, we're looking to move _away_ from collecting wrapper scripts.  Would you mind inlining this into the instructions?